Server/JokesOnYou.Web.Api/Controllers/UserController.cs

```diff
@@ -21,6 +21,13 @@ public UserController(IUserService userService)
             _userService = userService;
         }
 
+        [HttpGet("GetAll")]
+        public async Task<ActionResult<IEnumerable<AllUsersDTO>>> GetAllUsers()
```

Comment:
You can remove the GetAll and instead pluralize the controller name so it will be a Get request to api/Users

Reply:
So just [HttpGet("api/Users")]?

Reply:
You dont have to specify anything inside the HttpGet().

On top of our controller we have Route("api/[controller]") so it will automatically be a get request to the url api/Users (you have to change the controller name to Users though)